microsoft-edge/devtools-guide-chromium/whats-new/2024/09/devtools-129.md

```diff
@@ -0,0 +1,141 @@
+---
+title: What's New in DevTools (Microsoft Edge 129)
+description: "Updated labels for selector stats. Fixed whitespace in Activity Bar right-click menus. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 09/20/2024
+---
+# What's New in DevTools (Microsoft Edge 129)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Improvements to breakpoints
+
+<!-- Subtitle: Refreshed icons to help differentiate breakpoint types and the conditional breakpoint dialog alignment has been fixed.-->
+
+Logpoint and conditional breakpoints now have unique colors and icons to help you differentiate them from regular breakpoints in the line number column.  Previously, all types of breakpoints looked the same, with only the differentiating icon inline with the code.
+
+Before:
+
+![Old breakpoint icons](./devtools-129-images/old-breakpoints.png)
+
+After:
+
+![New breakpoint icons](./devtools-129-images/new-breakpoints.png)
+
+The dialog that's displayed for setting a conditional breakpoint has been fixed to prevent the width of the textbox from growing and causing alignment issues.
+
+Before:
+
+![Old conditional breakpoint dialog](./devtools-129-images/old-dialog.png)
+
+After:
+
+![New conditional breakpoint dialog](./devtools-129-images/new-dialog.png)
+
+
+<!-- ====================================================================== -->
+## Bug fixes in the Elements tool
+
+<!-- Screen readers will announce the "Show element in the Elements panel" button, and the context menu has been restored. -->
+
+
+<!-- ------------------------------ -->
+#### Screen readers announce the "Show element in the Elements panel" button
+
+Under the **Layout** pane,<!-- todo: in the Layout tab, --> screen readers now announce the **Show element in the Elements panel** button.<!-- todo: not found in UI.  add screenshot: -->
```

Comment:
need screenshot, can't find "Show element in the Elements panel" button 
![image](https://github.com/user-attachments/assets/6727ad4c-94c8-4a94-9500-95b51f2009f4)